PlanetClassExample/Program.cs

```diff
@@ -27,5 +27,11 @@ static void Main(string[] args)
 
             ship2.GetShipName();
         }
+
+
+        public void NewMethod()
+        {
+            System.Console.WriteLine("Does something cool!");
+        }
```

Comment:
Does not follow method naming convention.

Reply:
Yeah it doesn't